data/templates/postfix/main.cf

```diff
@@ -167,4 +171,20 @@ default_destination_rate_delay = 5s
 # By default it's possible to detect if the email adress exist
 # So it's easly possible to scan a server to know which email adress is valid
 # and after to send spam
-disable_vrfy_command = yes
\ No newline at end of file
+disable_vrfy_command = yes
+
+{% if relay_user != "" %}
+# Relay email through an other smtp account
+# enable SASL authentication
+smtp_sasl_auth_enable = yes
+# disallow methods that allow anonymous authentication.
+smtp_sasl_security_options = noanonymous
+# where to find sasl_passwd
+smtp_sasl_password_maps = hash:/etc/postfix/sasl_passwd
+{% if relay_port == "587" %}
+# Enable STARTTLS encryption
+smtp_use_tls = yes
```

Comment:
Is this what we want to do ? The documentation says

> With Postfix 2.3 and later use `smtp_tls_security_level` instead. 

 (we are using postfix 3.1 on stretch)

Also I'm confused by this setting, it sounds like it's really not related to just relaying ... Shouldn't we think about what setting we want in general ? (Default seems to be to send emails in cleartext between servers, maybe we want at least `smtp_tls_security_level = may` to encrypt when possible ?

Reply:
You are right smtp_tls_security_level = may replace this settings !